CHANGES.md

```diff
@@ -11,7 +11,14 @@ after the corresponding tracker issues.
 
 ## Changes since the last release
 
-- none so far
+- landmarks: When computing preferred operators, the `lmcount` heuristic
+  now considers all landmarks that have to be achieved in the future,
+  not only those that have not yet been accepted. This speeds up the
+  first iteration of LAMA with preferred operators and leads to an 
+  increased coverage on the IPC benchmarks, but does not affect other 
+  statistics (e.g., plan quality) significantly. (Holds also for the 
+  full anytime LAMA.) 
```

Comment:
Is this change log entry up to date? You wrote on the tracker that there are no more semantic changes.